AirshipGimbalAdapter.podspec

```diff
@@ -1,18 +1,18 @@
 
 Pod::Spec.new do |s|
   s.version                 = "4.0.0"
```

Comment:
Is this the version of the library? Should it correspond to the Changelog version 1.0.0?